engine/src/flutter/docs/contributing/Setting-up-the-Engine-development-environment.md

```diff
@@ -42,31 +42,23 @@ Run the following steps to set up your environment:
 >   * `GYP_MSVS_OVERRIDE_PATH=C:\Program Files\Microsoft Visual Studio\2022\Community`
 >     * Use the path of your installation.
 
-Create a new directory to hold the source code and move into it. Here, we are using the "engine" directory.
+Clone the Flutter source code. As of late 2024, the engine source is part of the main [flutter/flutter repo](https://github.com/flutter/flutter). The convention is to fork this repo and point `origin` to your fork and `upstream` to `git@github.com:flutter/flutter.git`.
 ```sh
-mkdir engine; cd engine;
+git clone git@github.com:<YOUR GITHUB USERNAME>/flutter.git
```

Comment:
This is maybe wrong because we need tags. Flutter has this documented as:

git clone flutter/flutter
git remote rename origin upstream
git remote add <your fork> as origin

Should we link to that?

Reply:
Yes good call, I've updated it to point to that doc and removed the bash commands.